pyquil/device/_isa.py

```diff
@@ -80,19 +109,17 @@ def to_dict(self):
             }
 
         :return: A dictionary representation of self.
-        :rtype: Dict[str, Any]
         """
 
-        def _maybe_configure(o, t):
-            # type: (Union[Qubit,Edge], str) -> dict
+        def _maybe_configure(o: Union[Qubit, Edge], t: str) -> Dict[str, Optional[Any]]:
```

Comment:
is `Optional` required / useful here? does just `Any` also include the possibility of `None`?